### ISSUE TITLE
AudioCueEditor Feature Suggestion: More Robust Loop Editing

### ISSUE THREAD

**Issue by MikoMMM** [LazyBone152/XV2-Tools#40]
something like literally changing the numbers of the loop start and ends like Eternity Audio Tools
While dragging the points is cool and all, it also tends to be incredibly unprecise
Example: something like this + dragging the loop start and end points
![image](https://user-images.githubusercontent.com/33367804/151718773-226c113f-70b4-4643-a757-e687f6152c8c.png)

**Comment by LazyBone152** [comment]
Yeah that could be added. Never really done any loop stuff myself so this feature is a little lacking right now.

I've also noticed in the past that the loop points themselves aren't 100% accurate, at least with respect to the playback within ACE.

**Comment by MikoMMM** [comment]
> I've also noticed in the past that the loop points themselves aren't 100% accurate, at least with respect to the playback within ACE. 

Yeah that's also an issue unfortunately

**Comment by LazyBone152** [comment]
Better looping features was implemented in 0.9. 

The inaccurate playback remains however.
